### ISSUE TITLE
Does WANDB save the architecture and weights?

### ISSUE THREAD

**Issue by shamoons** [wandb/client#876]
`wandb --version && python --version && uname`

* Weights and Biases version: 0.8.27
* Python version: 3.6.10+
* Operating System: MacOS

### Description
Pretty much as stated

**Comment by mfts** [comment]
Hi @shamoons,

the python package `wandb` can save your model file, log anything you'd like to save (like loss or accuracy numbers) and optionally also the hyperparameters you chose for a particular run.

1. Add wandb initializer at the beginning of your training script.
```
wandb.init()
``` 

2. if you would like to save your hyperparameters, just add them to the wandb configuration, eg:
```
wandb.config.episodes = 200
wandb.config.batch_size = 32
wandb.config.learning_rate = 0.003
...
```
3. Inside the training loop you can **log** anything you like but most often it will be a loss variable, i.e.: 
```
wandb.log({'Test Loss': test_loss})
```

4. and finally to save your `model.h5` file just run 
```
wandb.save(filename)
```

There are particular helper functions that you can use depending on the deep learning framework you use. For further information please have a look at the following wandb examples of these frameworks: 
- Pytorch: https://docs.wandb.com/library/example-projects/pytorch
- Keras: https://docs.wandb.com/library/example-projects/keras
- Tensorflow: https://docs.wandb.com/library/example-projects/tensorflow

Let me know if you have any other question. :)

**Comment by cvphelps** [comment]
@mfts Thanks for the amazing answer :)

**Comment by ariG23498** [comment]
Hey @shamoons 
We are closing this issue as the provided reply by @mfts makes sense and is the probable solution to the ticket.
Please feel free to reopen it for futher queries related to this ticket.
